src/components/Plugins.vue

```diff
@@ -59,7 +59,16 @@ const officalPlugins = computed(() => {
   }
 })
 
+
+const searchParams = new URLSearchParams(window.location.search);
+const query = searchParams.get("query")
+if (query) {
+  search.value = query
+}
+
 const communityPlugins = computed(() => {
+  window.history.pushState(null, null, search.value ? `?query=${search.value}`: '')
```

Comment:
I think we should place this effect in a `watchEffect` instead of inside a computed